site/ja/tutorials/text/text_generation.ipynb

```diff
@@ -0,0 +1,1256 @@
+        "And show his hands of life be proved to stand.\n",
+        "\n",
+        "PETRUCHIO:\n",
+        "I say he look'd on, if I must be content\n",
+        "To stay him from the fatal of our country's bliss.\n",
+        "His lordship pluck'd from this sentence then for prey,\n",
+        "And then let us twain, being the moon,\n",
+        "were she such a case as fills m\n",
+        "</pre>\n",
+        "\n",
+        "いくつかは文法にあったものがある一方で、ほとんどは意味をなしていません。このモデルは、単語の意味を学習していませんが、次のことを考えてみてください。\n",
+        "\n",
+        "* このモデルは文字ベースです。訓練が始まった時に、モデルは英語の単語のスペルも知りませんし、単語がテキストの単位であることも知らないのです。\n",
+        "\n",
+        "* 出力の構造は戯曲に似ています。だいたいのばあい、データセットとおなじ大文字で書かれた話し手の名前で始まっています。\n",
+        "\n",
+        "* 以下に示すように、モデルはテキストの小さなバッチ（各100文字）で訓練されていますが、一貫した構造のより長いテキストのシーケンスを生成できます。"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "srXC6pLGLwS6"
+      },
+      "source": [
+        "## 設定"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "WGyKZj3bzf9p"
+      },
+      "source": [
+        "### TensorFlow 等のライブラリインポート"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab_type": "code",
+        "id": "yG_n40gFzf9s",
+        "colab": {}
+      },
+      "source": [
+        "from __future__ import absolute_import, division, print_function, unicode_literals\n",
+        "\n",
+        "try:\n",
+        "  # %tensorflow_version は Colab 上でのみ利用可能\n",
+        "  %tensorflow_version 2.x\n",
+        "except Exception:\n",
+        "  pass\n",
+        "import tensorflow as tf\n",
+        "\n",
+        "import numpy as np\n",
+        "import os\n",
+        "import time"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "EHDoRoc5PKWz"
+      },
+      "source": [
+        "### シェイクスピアデータセットのダウンロード\n",
+        "\n",
+        "独自のデータで実行するためには下記の行を変更してください。"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab_type": "code",
+        "id": "pD_55cOxLkAb",
+        "colab": {}
+      },
+      "source": [
+        "path_to_file = tf.keras.utils.get_file('shakespeare.txt', 'https://storage.googleapis.com/download.tensorflow.org/data/shakespeare.txt')"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "UHjdCjDuSvX_"
+      },
+      "source": [
+        "### データの読み込み\n",
+        "\n",
+        "まずはテキストをのぞいてみましょう。"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab_type": "code",
+        "id": "aavnuByVymwK",
+        "colab": {}
+      },
+      "source": [
+        "# 読み込んだのち、Python 2 との互換性のためにデコード\n",
```

Comment:
This translation is pretty good!

- Original: `Read, then decode for py2 compat.` <- unnecessasry abbreviation...

Reply:
Thank you.